Apps/W1/HybridGP/app/src/Migration/GPTables/GPPopulateCombinedTables.Codeunit.al

```diff
@@ -323,11 +331,10 @@ codeunit 40125 "GP Populate Combined Tables"
         GPRM00101: Record "GP RM00101";
         GPRM00103: Record "GP RM00103";
         GPSY01200: Record "GP SY01200";
-        GPCompanyMigrationSettings: Record "GP Company Migration Settings";
```

Comment:
Should we obsolete this table / fields on the table if they are not used?

Reply:
I will look into this for the next PR